src/test/java/com/atlantbhinternship/auctionapp/models/ConfirmationTokenTest.java

```diff
@@ -0,0 +1,4 @@
+import static org.junit.jupiter.api.Assertions.*;
+class ConfirmationTokenTest {
```

Comment:
Should not add empty classes. You should provide basic implementation within class or remove.